### ISSUE TITLE
Line Chart Tooltip with 1 point error

### ISSUE THREAD

**Issue by frankleib** [radzenhq/radzen-blazor#559]
If I have in a LineSeries in a line chart only 1 point and I hover over it, the tool tip is not shown

```
 <RadzenChart @ref="chart0">
        @foreach(var item in delevererEKDatas!)
        {
            string tile = item.LineName;
                string color = getRandomColor(item.MainDeliverer);
                <RadzenLineSeries Stroke="@color" Smooth="@smooth" Data="@item.EKData" CategoryProperty="Date" Title=@tile LineType="LineType.Solid" ValueProperty="Price">
                    <ChildContent>
                        <RadzenMarkers MarkerType="MarkerType.Auto" Stroke="@color" />
                    </ChildContent>
                    <TooltipTemplate Context="data">
                    <div>
                        @data.DelInfo<br/>
                        <strong>@data.Date.ToString("dddd, dd MMMM yyyy")</strong><br/>
                        <strong>EK-Preis: @data.Price.ToString("N2")</strong><br/>
                        ab Menge @data.MinQuantity.ToString("N0")<br/>
                            @if (@data.AvailableQuantity is not null) 
                            { 
                                <span>verfügbare Menge: @data.AvailableQuantity</span>
                            }
                    </div>
                    </TooltipTemplate>
                </RadzenLineSeries>
            }
    
        <RadzenCategoryAxis  Padding="20" Min="@searchArticleDateDeliverer.StartDate" Max="@searchArticleDateDeliverer.EndDate" Step="31" FormatString="{0:dd.MM.yy}" />
        <RadzenValueAxis Formatter="@FormatAsEURO">
            <RadzenGridLines Visible="true" />
        </RadzenValueAxis>
        @if(delevererEKDatas.Count == 1)
        {
                <RadzenLegend Position="LegendPosition.Bottom" Visible="false" />
        }
        else if(delevererEKDatas.Count > 4)
        {
            <RadzenLegend Position="LegendPosition.Right" Visible="true" />
        }
        else
        {
            <RadzenLegend Position="LegendPosition.Bottom" Visible="true" />
        }
    </RadzenChart> 
```

**Comment by akorchev** [comment]
Closed by 1d357aa

**Comment by frankleib** [comment]
@akorchev hi, it works? You don't give a comment i. your close... :) Thanks 4 u fast checks. I use blazor as server app, is this the prob?
